test/build_defs/test.build_defs

```diff
@@ -23,13 +23,13 @@ def please_repo_e2e_test(
     ]
 
     if expected_output:
-       test_cmd += [f"$TOOLS_CONTENT_CHECKER '{o}' '{c}'" for o, c in expected_output.items()]
+        test_cmd += [f"$TOOLS_CONTENT_CHECKER '{o}' '{c}'" for o, c in expected_output.items()]
```

Comment:
Not the most elegant but gets the job done in a POSIX compliant way. This change was required to make newlines in multiline strings not to cause grep to return false positives.